src/player/player.js

```diff
@@ -17,26 +17,25 @@ export class Player extends Phaser.Sprite {
     this.anchor.setTo(0.5, 0);
     this.scale.setTo(this.config.scale, this.config.scale);
     this.animations.add('run', [1, 2, 3, 4, 5], 13, true);
-    game.add.existing(this);
 
     this.health = health;
     this.maxHealth = maxHealth;
     this.speed = speed;
-    this.fallVelocity = 0;
     this.direction = 'right';
-    
+
     this.controlsSetup();
     this.render();
+
+    game.add.existing(this);
   }
 
   /**
    * Setup sprite body settings
    */
   bodySetup () {
-    this.game.physics.enable([this], Phaser.Physics.ARCADE);
-    this.body.gravity.y = 20000;
-    this.body.bounce.y = 0.2;
-    this.body.collideWorldBounds = true;
+    // this.fixedRotation = true;
+    // this.damping = 0.2;
+    // this.collideWorldBounds = true;
```

Comment:
I know these are useless but they are reminders for  next steps for me to figure out how to apply specific Physics rules to objects.